keyboards/mechmini/v2/keymaps/via/keymap.c

```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FN1 1
+#define _FN2 2
+#define _FN3 3
+
```

Comment:
```suggestion
```

Reply:
Since the names are not used for the layer switching, I'd get rid of them altogether.
It's not like they have too much reason in `via` keymaps to begin with.